crypto/cardano-sl-crypto.cabal

```diff
@@ -27,6 +27,7 @@ library
                        Pos.Crypto.Signing
                        Pos.Crypto.Signing.Redeem
                        Pos.Crypto.Signing.Safe
+                       Pos.Crypto.Signing.Types.Safe
```

Comment:
Same thing as about exposing `Pos.Core.Update`. This one is re-exported by `Pos.Crypto.Signing.Safe`.